cmd/draino/draino.go

```diff
@@ -60,7 +60,7 @@ func main() {
 
 		protectedPodAnnotations = app.Flag("protected-pod-annotation", "Protect pods with this annotation from eviction. May be specified multiple times.").PlaceHolder("KEY[=VALUE]").Strings()
 
-		conditions = app.Arg("node-conditions", "Nodes for which any of these conditions are true will be cordoned and drained.").Required().Strings()
+		conditions = app.Arg("node-conditions", "Nodes for which any of these conditions are true will be cordoned and drained. <TYPE[=STATE]>").Strings()
```

Comment:
Please use `.PlaceHolder("TYPE[=STATUS]").` here rather than including the placeholder in the help text.

Do we really want to remove the `Required()` option here? Doing so will make it possible to start Draino without specifying any node conditions, a reversion of #31.